src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/PaymentAccountServiceImpl.java

```diff
@@ -61,4 +69,63 @@ public Organisation findPaymentAccountsByEmail(String email) {
         }
         return organisation;
     }
-}
+
+    @Transactional

```

Comment:
You should leave a note that explains way only the super-user is being checked for user_account_maps as technically this is a defect and if corrected this logic may need to then change in the future.